inde.html

```diff
@@ -0,0 +1,99 @@
+
+</html>
+<html class=""><head>
+    <link rel="stylesheet" href="style.css">
+  <style id="__web-inspector-hide-shortcut-style__">
+.__web-inspector-hide-shortcut__, .__web-inspector-hide-shortcut__ *, .__web-inspector-hidebefore-shortcut__::before, .__web-inspector-hideafter-shortcut__::after
+{
+    visibility: hidden !important;
+}
+</style></head>
+  <body>
+    <div class="container">
+      <div class="header">
+        <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
```

Comment:
Kindly make this a header tag instead of a div tag. It is more semantic to use a header tag so that it defines the meaning of the content they contain..